app/views/shared/navigation/_all_projects_primary_navigation.html.erb

```diff
@@ -1,44 +1,27 @@
 <div class="moj-primary-navigation">
-
```

Comment:
Is there a rationale for these re-formatting changes?

Reply:
> Is there a rationale for these re-formatting changes?

thanks for the question: I've now broken out the reformatting into a separate commit with an explanation of why it's done: https://github.com/DFE-Digital/dfe-complete-conversions-transfers-and-changes/pull/2108/commits/83798cd5f830bbe81d994e13d42ac81257ec5e5b

Reply:
Thanks for that, makes sense!